src/ServerConfig.php

```diff
@@ -61,6 +61,12 @@ class ServerConfig
 		'server.type'         => "nginx",
 		'session.handler'     => "database",
 		'session.lifetime'    => "15",
+		'network.name'		  => "joomla",
+		'selenium.no'		  => "1",
+		'selenium.version'	  => "latest",
+		'weblinks.repoOwner'  => "joomla",
+		'weblinks.repoName'   => "weblinks",
+		'weblinks.repoBranch' => "container-test",
```

Comment:
Better use `extension.repoOwner`, `extension.repoName`, and `extension.repoBranch` to keep it independent from WebLinks..